playground/src/workbench/prql-syntax.js

```diff
@@ -14,13 +14,14 @@ const TRANSFORMS = [
 ];
 const BUILTIN_FUNCTIONS = ["switch"]; // "in", "as"
 const KEYWORDS = ["func", "table", "prql"];
-const LITERALS = ["and", "or", "not", "null", "true", "false"];
+const LITERALS = ["null", "true", "false"];
+const OPERATORS = ["and", "or"]; // "not"
```

Comment:
Thanks for changing this. Shall we merge with the `operators` here? https://github.com/PRQL/prql/pull/1447/files/80d7c57262bda9e0d3d6997561e99c6e55111e71..28cd9b487b0fc87d34bf75dd7d36e91247a09b3a#diff-90def8ff0b4c8f49895df5b4685919259d365a97481f30bdda177b2242066cc4R26